config/vufind/PAIA.ini

```diff
@@ -18,6 +18,25 @@ baseUrl = ""
 ; same as for DAIA cache (see cacheLifetime setting in DAIA.ini).
 ;paiaCache = false
 
+; VuFind allows to report account blocks to the user. As PAIA blocks implicitly by
+; not issuing PAIA scopes for a patron VuFind would report any possibly missing
+; scope. Therefore with the following setting you can define for which missing scope
+; the user will get a notification in MyResearch views. By default any missing scope
+; results in a notification.
+; For a comprehensive list of PAIA scopes refer to
+; http://gbv.github.io/paia/paia.html#access-tokens-and-scopes .
+;muteAccountBlocksNotificationForScopes[] = "read_patron"
```

Comment:
Would it be better to make this "opt-in" rather than "opt-out" -- that is, rather than alerting for missing scopes unless they are muted, instead alert only for scopes where alerts are requested? This seems a little safer, since otherwise depending on usage of a system, users might end up getting a lot of false positive block requests. Also, making it opt-in might make it easier to add custom scopes (or newly added ones in the future).

In fact, you could even customize the text of the block messages directly in the config file... what if you used a format like:

`addBlockNotificationForMissingScope[update_patron] = "ILSMessages::no_update_patron_scope"`

This approach might enable you to reduce the amount of code in the driver as well! There seem to be some special cases -- e.g. around patron scopes -- but I still imagine some reduction may be possible.

(Just brainstorming, though -- feel free to ignore the suggestion if you disagree).